examples/lists/Cargo.toml

```diff
@@ -5,5 +5,5 @@ authors = ["seladb <pcapplusplus@gmail.com>"]
 edition = "2018"
 
 [dependencies]
-pickledb = { path = "../../" }
+pickledb = { path = "../../", default-features = false, features = ["bincode"] }
```

Comment:
why do we want to add the `bincode` feature implicitly and not use `json` as the default feature?

Reply:
Not sure what you mean. `json` is the default feature (disabled to reduce space), and `bincode` is used here because that's the serialization method used in the example.

Reply:
Oh you're right, I got confused, sorry about that...